components/GridView.vue

```diff
@@ -89,16 +91,20 @@ async function validateAndSaveLayout(newLayout: string): Promise<void> {
     return
   }
 
-  if (!isValidLayout(parsedLayout)) {
+  // Convert the layout to grid layout items
+  const gridLayoutItems = toGridLayoutItems(parsedLayout, COL_NUM_LARGE)
+
+  if (!isValidLayout(gridLayoutItems)) {
     alert('Invalid schema 😡')
```

Comment:
Let's use `console.warn` instead, we should load default if this happens I guess.